app/api/submission/portfolio_api.rb

```diff
@@ -74,11 +77,13 @@ class PortfolioApi < Grape::API
 
         evidence_loc = project.portfolio_path
 
-        if evidence_loc.nil? || File.exist?(evidence_loc) == false
-          evidence_loc = Rails.root.join('public', 'resources', 'FileNotFound.pdf')
-          header['Content-Disposition'] = 'attachment; filename=FileNotFound.pdf'
-        else
-          header['Content-Disposition'] = 'attachment; filename=portfolio.pdf'
+        if params[:as_attachment]
+          if evidence_loc.nil? || File.exist?(evidence_loc) == false
```

Comment:
We need to pull this test out of the attachment test. This checks if the file does not exist and switches it to use the FileNotFound.pdf instead. This way the end point always returns some PDF.